src/main/java/com/ft/methodecontentcollectionmapper/client/DocumentStoreApiClient.java

```diff
@@ -0,0 +1,112 @@
+package com.ft.methodecontentcollectionmapper.client;
+
+import com.ft.methodecontentcollectionmapper.configuration.UppServiceConfiguration;
+import com.ft.methodecontentcollectionmapper.exception.TransientUuidResolverException;
+import com.ft.methodecontentcollectionmapper.exception.UuidResolverException;
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientHandler;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+import com.sun.jersey.api.client.filter.ClientFilter;
+import com.sun.jersey.client.apache4.ApacheHttpClient4Handler;
+import io.dropwizard.setup.Environment;
+import org.apache.http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.core.UriBuilder;
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URLEncoder;
+
+import static com.ft.api.util.transactionid.TransactionIdUtils.TRANSACTION_ID_HEADER;
+
+/**
+ * Copied from methode-list-mapper.
+ */
+public class DocumentStoreApiClient extends UppServiceClient {
+    private static final Logger LOG = LoggerFactory.getLogger(DocumentStoreApiClient.class);
+    private static final String QUERY_PATH = "/content-query";
+    private static final String LISTS_PATH = "/lists";
+
+    private final URI queryEndpoint;
+    private final URI listsEndpoint;
```

Comment:
This field isn't used and should be removed.

Reply:
fixed.